lib/staccato/V4/tracker.rb

```diff
@@ -19,12 +19,14 @@ def initialize(
       api_secret,
       client_id = nil,
       user_id = nil,
+      timestamp_micros = nil,
```

Comment:
Note to reviewers : Keeping positional arguments in order. We pass the below from stargate codebase

```
    Staccato::V4::Tracker.new(
      options.fetch('measurement_id'),
      options.fetch('api_key'),
      client_id,
      options.fetch('user_id'),
      timestamp_micros
    )
```